syntaxes/language-configuration.json

```diff
@@ -0,0 +1,14 @@
+{
+  "wordPattern": "[a-zA-Z0-9_-]+",
+  "brackets": [
+    ["{", "}"],
+    ["[", "]"],
```

Comment:
Where do you expect those characters in COBOL - they are invalid outside of strings and won't be matched inside, no?